src/test/test-math-util.c

```diff
@@ -54,7 +54,14 @@ TEST(iszero_safe) {
         assert_se( iszero_safe(DBL_MIN / -INFINITY));
         assert_se( iszero_safe(DBL_MAX / INFINITY / 2));
         assert_se( iszero_safe(DBL_MAX / -INFINITY * DBL_MAX));
+#if defined(__i386__) && !defined(__SSE_MATH__)
```

Comment:
One remaining edge case: if SSE2 instructions are not enabled, `-mfpmath=sse` only works for single-precision floats.
```suggestion
#if defined(__i386__) && (!defined(__SSE_MATH__) || !defined(__SSE2__))
```
I have tested this locally and it works as expected.

Reply:
Maybe use `__SSE2_MATH__` for that case?

Reply:
Ah, I wasn't aware of that one. Tested locally and it also works.
```suggestion
#if defined(__i386__) && !defined(__SSE2_MATH__)
```